ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/BlankApp/BlankApp.UWP/BlankApp.UWP.csproj

```diff
@@ -12,8 +12,8 @@
     <AssemblyName>BlankApp.UWP</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.15063.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.10586.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.10586.0</TargetPlatformVersion>

```

Comment:
This version should probably target 16299, or at a minimum 14393 as the TargetPlatformVersion (as 10596 isn't supported any more)

Reply:
@SimonDarksideJ - i suppose the downside to moving to 16299 is users are forced to update to the newest SDK?  It seems like to use it you have to be on VS2017 which might be annoying to existing users.

14393 seems better in that regard and users can always update their project to move forward.

I think @SimonDarksideJ knows more about this then me... so i would go with his recommendation here.

Reply:
Not really, this simply states the requested version is MS's currently supported OS version.
But the minimum is the lowest level current API that is supported.
We do need to keep a track of SDK's that MS actively supports but since MG doesn't actually use many native API's we should be covered internally, the consumer can set their versions to whatever they want afterwards.